test/test_repository/test_project_repository.py

```diff
@@ -183,12 +183,15 @@ def test_save(self):
 
     @with_context
     def test_save(self):
-        """Test save fails if the project has no password"""
+        """Test save does not fail if the project has no password"""
```

Comment:
Should we include another test that checks if `PROJECT_PASSWORD_REQUIRED is True` then it should succeed?